src/translation/translation_nb_NO.ts

```diff
@@ -2730,7 +2730,7 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
     <message>
         <location filename="../serverdlg.cpp" line="245"/>
         <source>&amp;Hide %1 server</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
That's strange: Why is it an empty string?

@hoffie probably some future translation CI script should validate that too.

Reply:
It could be flagged "read only" in Weblate to avoid it for now.

Reply:
@comradekingu: you just seem to have forgotten to translate this string, don't you?

Reply:
@trebmuh 
https://hosted.weblate.org/translate/jamulus/jamulus-app/nb_NO/?checksum=bd82c5a091ec5c9c
From memory I was unsure if "%1" is a numeral or the name of a server.
Hence the question https://hosted.weblate.org/translate/jamulus/jamulus-app/nb_NO/?checksum=bd82c5a091ec5c9c
"Is this a known server (name), or is it an amount of servers?"

Reply:
Then you decided to leave it in English?
If so, that's a better idea IMHO to leave the translation "empty" then it's still marked "unfinished" and fall back to display it in English.